R/umap.R

```diff
@@ -7,7 +7,7 @@
 ##' @param min.dist Controls how tightly the embedding is allowed compress points together.
 ##' @param seed Random seed.
 ##'
-##' @importFrom reticulate import
+##' importFrom reticulate import
```

Comment:
I did this so that it doesn't appear in the check and DESCRIPTION

Reply:
So why is it ok to remove `reticulate`? If its not used can't we just delete line 10 altogether?

Reply:
The whole file is commented out. Should we just remove the file?

I modified it so that if someone wanted to use this file again, they could uncomment it and add the @ to make the import work again.

Reply:
I see. I didn't realise it was all commented out. I guess we should just leave it then with the change you made to remove it from check and DESCRIPTION